content/en/docs/spinnaker-install-admin-guides/prometheus-monitoring.md

```diff
@@ -53,7 +61,190 @@ Access the Prometheus web interface by using the `kubectl port-forward` command.
+
+management:
+  endpoints:
+    web:
+      # READ THE SECURITY WARNING ABOVE ON THIS!
+      exposure.include: health,info,aop-prometheus
+spinnaker:
+  extensibility:
+    plugins:
+      Armory.ObservabilityPlugin:
+        # THIS Is absolutely required, though potentially redundant, otherwise the plugin won't start
+        enabled: true
+        version: 1.0.0
+        # This is the basic configuration for prometheus to be enabled
+        config.metrics:
+          prometheus:
+            enabled: true
+    repositories:
+      armory-observability-plugin-releases:
+        url: https://raw.githubusercontent.com/armory-plugins/armory-observability-plugin-releases/master/repositories.json
+```
+More options for management endpoints and the plugin are available on the plugin homepage.
+
+
+####  Grant prometheus permissions 
+Add permissions for Prometheus by applying the following configuration to your cluster.  This is documented
+in more depth at the [Prometheus Operator homepage](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/rbac.md).  
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: prometheus
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: prometheus
+subjects:
+  - kind: ServiceAccount
+    name: prometheus
+    namespace: monitoring
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: prometheus
+rules:
+- apiGroups: [""]
+  resources:
+  - nodes
+  - nodes/metrics
+  - services
+  - endpoints
+  - pods
+  verbs: ["get", "list", "watch"]
+- apiGroups: [""]
+  resources:
+  - configmaps
+  verbs: ["get"]
+- nonResourceURLs: ["/metrics"]
+  verbs: ["get"]
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: monitoring
+  name: prometheus
+```
+
+#### Add the ServiceMonitor
+Prometheus Operator uses a "ServiceMonitor" to add targets to be scraped for monitoring.  Below is a sample
+ showing how to monitor pods which are using the plugin to expose the aop-prometheus endpoint.  Note this has
+  both exclusion of certain services (aka redis) and changes the gate endpoint to show different options.  
+
+These are ONLY examples of potential configurations and though they work it is recommended that you understand how 
+they operate and find services.  You'll want to change these settings and adapt them to your environment.
+
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: spin
+    release: prometheus-operator
+  name: spinnaker-all-metrics
+  namespace: spinnaker
+spec:
+  endpoints:
+  - interval: 10s
+    path: /aop-prometheus
+  selector:
+    matchExpressions:
+    - key: cluster
+      operator: NotIn
+      values:
+      - spin-gate
+      - spin-gate-api
+      - spin-gate-custom
+      - spin-deck
+      - spin-deck-custom
+      - spin-redis
+      - spin-terraformer
```

Comment:
@jasonmcintosh Need to add `spin-dinghy` to this list.

Reply:
Should be done ;)